<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>比图吧 - Untitle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/MainIco.png</normaloff>:/img/MainIco.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea_GraphSelect">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>876</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>基础图形</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="chart_ellipse">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/Round.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="chart_roundrect">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/Rounded_Rectangle.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="chart_diam">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/diamond.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="chart_rect">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/rectangle.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="chart_trapezoid">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/trapezoid.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="chart_line">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/img/line.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>UML通用</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>500</height>
           </size>
          </property>
          <property name="title">
           <string>UML用例图</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="StylePanel" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QGroupBox" name="FillColorGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>填充颜色：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="FillColorButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QGroupBox" name="LineWidthGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>线宽：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="LineWidthCBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>1px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9px</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10px</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="LineColorGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>线条颜色：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LineColorButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QGroupBox" name="LineStartStyleGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>起点样式：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="LineStartStyleCBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow0.png</normaloff>:/img/arrow0.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow1.png</normaloff>:/img/arrow1.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow2.png</normaloff>:/img/arrow2.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow3.png</normaloff>:/img/arrow3.png</iconset>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="6">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <widget class="QGroupBox" name="LineEndStyleGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>终点样式：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="LineEndStyleCBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow0.png</normaloff>:/img/arrow0.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow1.png</normaloff>:/img/arrow1.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow2.png</normaloff>:/img/arrow2.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/img/arrow3.png</normaloff>:/img/arrow3.png</iconset>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_Painter">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_main">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>2018</width>
            <height>1018</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="FlowChart" name="flowChart_widget" native="true">
             <property name="minimumSize">
              <size>
               <width>2000</width>
               <height>1000</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>比图吧 - Untitle</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSaveFile"/>
   <addaction name="actionNewFile"/>
   <addaction name="actionSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetting">
   <property name="icon">
    <iconset theme="Setting" resource="res.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file_open.png</normaloff>:/img/file_open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file_add.png</normaloff>:/img/file_add.png</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file_delete.png</normaloff>:/img/file_delete.png</iconset>
   </property>
   <property name="text">
    <string>关闭文件</string>
   </property>
   <property name="toolTip">
    <string>关闭文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file_save.png</normaloff>:/img/file_save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FlowChart</class>
   <extends>QWidget</extends>
   <header location="global">flowchart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
